src/Thinktecture.Blazor.ViewTransitions/README.md

```diff
@@ -0,0 +1,116 @@
+# Thinktecture.Blazor.ViewTransitions
+
+
+## Introduction
+This package should help you to use the ViewTransition API in your Blazor application. The package contains two ways to use the ViewTransition API.
+If you want to know how the View Transition API works look [here](https://drafts.csswg.org/css-view-transitions/#viewtransition).
+
+https://github.com/patrickjahr/pazor-view-transition-api/assets/16818441/424683ed-f30c-4b14-a4db-5d770c076027
+
+## Getting started
+
+### Prerequisites
+
+You need .NET 7.0 or newer to use this library.
+
+[Download .NET 7](https://dotnet.microsoft.com/download/dotnet/7.0)
+
+### Platform/Browser support
+
+Please note that ViewTransition API is not yet supported in all major browsers. [Here](https://caniuse.com/?search=ViewTransition%20API), you can find the current support.
+
+### Installation
+
+
+You can install the package via NuGet with the Package Manager in your IDE or alternatively using the command line:
+
+```
+dotnet add package Pazor.ViewTransitionApi
+```
+
+
+## Usage
+
+The package can be used in Blazor WebAssembly projects.
+
+### Add to service collection
+
+To make the `IViewTransitionService` available on all pages, register it at the IServiceCollection in `Program.cs` before the host is built:
```

Comment:
```suggestion
To make the IViewTransitionService available on all pages, register it at the IServiceCollection in `Program.cs` before the host is built:
```

Service names are sometimes used with backticks, sometimes without. I think like the usage _without_ backticks is the more common one in this repository.

Reply:
Looks like this change was not applied.